src/Microsoft.TestPlatform.Utilities/CodeCoverageRunSettingsProcessor.cs

```diff
@@ -0,0 +1,484 @@
+            @"      <CollectFromChildProcesses>True</CollectFromChildProcesses>",
+            @"      <CollectAspDotNet>false</CollectAspDotNet>",
+            @"      <SymbolSearchPaths />",
+            @"      <Functions>",
+            @"        <Exclude>",
+            @"          <Function>^std::.*</Function>",
+            @"          <Function>^ATL::.*</Function>",
+            @"          <Function>.*::__GetTestMethodInfo.*</Function>",
+            @"          <Function>.*__CxxPureMSILEntry.*</Function>",
+            @"          <Function>^Microsoft::VisualStudio::CppCodeCoverageFramework::.*</Function>",
+            @"          <Function>^Microsoft::VisualStudio::CppUnitTestFramework::.*</Function>",
+            @"          <Function>.*::YOU_CAN_ONLY_DESIGNATE_ONE_.*</Function>",
+            @"          <Function>^__.*</Function>",
+            @"          <Function>.*::__.*</Function>",
+            @"        </Exclude>",
+            @"      </Functions>",
+            @"      <Attributes>",
+            @"        <Exclude>",
+            @"          <Attribute>^System.Diagnostics.DebuggerHiddenAttribute$</Attribute>",
+            @"          <Attribute>^System.Diagnostics.DebuggerNonUserCodeAttribute$</Attribute>",
+            @"          <Attribute>System.Runtime.CompilerServices.CompilerGeneratedAttribute$</Attribute>",
+            @"          <Attribute>^System.CodeDom.Compiler.GeneratedCodeAttribute$</Attribute>",
+            @"          <Attribute>^System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute$</Attribute>",
+            @"          <Attribute>^Microsoft.VisualStudio.TestPlatform.TestSDKAutoGeneratedCode.*</Attribute>",
+            @"        </Exclude>",
+            @"      </Attributes>",
+            @"      <Sources>",
+            @"        <Exclude>",
+            @"          <Source>.*\\atlmfc\\.*</Source>",
+            @"          <Source>.*\\vctools\\.*</Source>",
+            @"          <Source>.*\\public\\sdk\\.*</Source>",
+            @"          <Source>.*\\externalapis\\.*</Source>",
+            @"          <Source>.*\\microsoft sdks\\.*</Source>",
+            @"          <Source>.*\\vc\\include\\.*</Source>",
+            @"          <Source>.*\\msclr\\.*</Source>",
+            @"          <Source>.*\\ucrt\\.*</Source>",
+            @"        </Exclude>",
+            @"      </Sources>",
+            @"      <CompanyNames/>",
+            @"      <PublicKeyTokens/>",
+            @"    </CodeCoverage>",
+            @"  </Configuration>",
+            @"</DataCollector>");
+        #endregion
+
+        /// <summary>
+        /// Represents the default settings loaded as an <see cref="XmlNode"/>.
+        /// </summary>
+        private XmlNode defaultSettingsRootNode;
+
+        /// <summary>
+        /// Represents a list of exclusion types tracked by this processor.
+        /// </summary>
+        private IEnumerable<Exclusion> exclusions;
```

Comment:
Please change it to IList<Exclusion>.
IEnumerable can be iterated only once. So it means that Process can be called once on the instance. Having it IEnumerable is not benefit in this case

Reply:
Done.